packages/stores/types.ts

```diff
@@ -188,12 +188,12 @@ type TButtonProps = {
     text: string;
 };
 
-type TNotificationMessage = {
-    action?: {
-        onClick: () => void;
-        route?: string;
-        text: string;
-    };
+export type TActionProps = TButtonProps & {
+    route?: string;
+};
+
+export type TNotificationMessage = {
```

Comment:
@shaheer-deriv Don't export types from this file 🙏🏻

Reply:
@shaheer-deriv In 99.99% of cases there is nothing as shared types 😅 Types should infer 😬

If these types are coming from stores, Keep them in `packages/stores/types.ts` but you don't need to export them. This type file is not a shared place to dump all the types and then import them, We are trying to make the stores type-safe using `useStore` hook.

Ideally, you shouldn't need to use the type anywhere else as the `useStore` hook will give you the data already type-safe and inferred. But still, if you need you can use TS generics to reach to the type. Something like;
```ts
type TLinkedTo = ReturnType<typeof useStore>['client']['accounts'][string]['linked_to'];
```